src/content/docs/en/guides/integrations-guide/mdx.mdx

```diff
@@ -84,13 +84,136 @@ For other editors, use the [MDX language server](https://github.com/mdx-js/mdx-a
 
 ## Usage
 
-With the Astro MDX integration, you can [add MDX pages to your project](/en/guides/markdown-content/#markdown-and-mdx-pages) by adding `.mdx` files within your `src/pages/` directory. You can also [import `.mdx` files](/en/guides/markdown-content/#importing-markdown) into `.astro` files.
+Visit the [MDX docs](https://mdxjs.com/docs/what-is-mdx/) to learn about using standard MDX features.
 
-Astro's MDX integration adds extra features to standard MDX, including Markdown-style frontmatter. This allows you to use most of Astro's built-in Markdown features like a [special frontmatter `layout` property](/en/guides/markdown-content/#frontmatter-layout).
+## MDX in Astro
 
-See how MDX works in Astro with examples in our [Markdown & MDX guide](/en/guides/markdown-content/).
+Adding the MDX integration enhances your Markdown authoring with JSX variables, expressions and components.
 
-Visit the [MDX docs](https://mdxjs.com/docs/what-is-mdx/) to learn about using standard MDX features.
+It also adds extra features to standard MDX, including support for Markdown-style frontmatter in MDX. This allows you to use most of [Astro's built-in Markdown features](/en/guides/markdown-content/).
+
+`.mdx` files must be written in [MDX syntax](https://mdxjs.com/docs/what-is-mdx/#mdx-syntax) rather than Astro’s HTML-like syntax.
+
+### Using Exported Variables in MDX
+
+MDX supports using `export` statements to add variables to your MDX content.
+
+For example, you can export a `title` field from an MDX page or component to use as a heading with `{JSX expressions}`:
+
+```mdx title="/src/pages/posts/post-1.mdx"
+export const title = 'My first MDX post'
+
+# {title}
+```
+
+MDX files can also use an `export` statement to export data to a component that imports it.
+
+For example, you can export a `title` field from an MDX page or component.
+
+```mdx title="/src/pages/posts/post-1.mdx"
+export const title = 'My first MDX post'
+```
+
+This `title` will be accessible from `import` and `import.meta.glob()` statements:
+
+```astro title="// src/pages/index.astro"
+---
+const matches = await import.meta.glob('./*.mdx', { eager: true });
+const posts = Object.values(posts);
+---
+
+{posts.map(post => <p>{post.title}</p>)}
+```
+
+#### Exported Properties
+
+The following properties are available to a `.astro` component when using an `import` statement or `import.meta.glob()`:
+
+- **`file`** - The absolute file path (e.g. `/home/user/projects/.../file.md`).
```

Comment:
nit: Since this is the MDX guide, I think we should replace the extension:
```suggestion
- **`file`** - The absolute file path (e.g. `/home/user/projects/.../file.mdx`).
```